README.md

```diff
@@ -1,91 +1,135 @@
+- [cuSpatial Developer Documentation](https://docs.rapids.ai/api/cuspatial/stable/developer_guide/index.html): Understand cuSpatial's architecture
+- [Getting Started](https://rapids.ai/start.html): Instructions for installing cuSpatial
+- [cuSpatial Community](https://github.com/rapidsai/cuspatial/discussions): Get help, collaborate, and ask the team questions
+- [cuSpatial Issues](https://github.com/rapidsai/cuspatial/issues/new/choose): Request a feature/documentation or report a bug
+- [cuSpatial Roadmap](https://github.com/orgs/rapidsai/projects/41/views/5): Report issues or request features.
 
-1. Spatial window query
-2. Point-in-polygon test
-3. Haversine distance
-4. Hausdorff distance
-5. Deriving trajectories from point location data
-6. Computing distance/speed of trajectories
-7. Computing spatial bounding boxes of trajectories
-8. Quadtree-based indexing for large-scale point data
-9. Quadtree-based point-in-polygon spatial join
-10. Quadtree-based point-to-linestring nearest neighbor distance
-11. Distance computation (point-point, point-linestring, linestring-linestring)
-12. Finding nearest points between point and linestring
+## Overview
+Now based on the [GeoArrow](https://github.com/geoarrow/geoarrow) format, cuSpatial accelerates vector geospatial operations through GPU parallelization. As part of the RAPIDS libraries, it's inherently connected to [cuDF](https://github.com/rapidsai/cudf), [cuML](https://github.com/rapidsai/cuml), and [cuGraph](https://github.com/rapidsai/cugraph), enabling GPU acceleration across entire workflows. 
 
-Future support is planned for the following operations:
+cuSpatial's Python API is closely matched to GeoPandas and data can seamlessly transition between the two:
+```python
+import geopandas
+from shapely.geometry import Polygon
+import cuspatial
 
-1. Temporal window query
-2. Temporal point query (year+month+day+hour+minute+second+millisecond)
-3. Grid-based indexing for points and polygons
-4. R-Tree-based indexing for Polygons/Polylines
+p1 = Polygon([(0, 0), (1, 0), (1, 1)])
+p2 = Polygon([(0, 0), (1, 0), (1, 1), (0, 1)])
+geoseries = geopandas.GeoSeries([p1, p2])
 
-## Install from Conda
-
-To install via conda:
-
-```shell
-conda install -c conda-forge -c rapidsai-nightly cuspatial
+cuspatial_geoseries = cuspatial.from_geopandas(geoseries)
+print(cuspatial_geoseries)
+```
+Output:
+```
+0    POLYGON ((0 0, 1 0, 1 1, 0 0))
+1    POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))
 ```
 
-## Install from Source
-
-To build and install cuSpatial from source:
-
-### Pre-requisite
-
-- gcc >= 7.5
-- cmake >= 3.23
-- miniconda
+For additional examples, browse our complete [API documentation](https://docs.rapids.ai/api/cuspatial/stable/), or check out our more detailed [notebooks](https://github.com/rapidsai/notebooks-contrib) - the NYC Taxi and Weather notebooks make use of cuSpatial.
```

Comment:
Avoid possessives / "royal we".

```suggestion
For additional examples, browse the complete [API documentation](https://docs.rapids.ai/api/cuspatial/stable/), or check out more detailed [notebooks](https://github.com/rapidsai/notebooks-contrib). the [NYC Taxi][1] and [Weather][2] notebooks make use of cuSpatial.
```

And add the long links to those two notebooks at the bottom as footnote links.
```
[1]:https://github.com/rapidsai-community/notebooks-contrib/blob/main/community_tutorials_and_guides/taxi/NYCTaxi-E2E.ipynb
[2]:https://github.com/rapidsai-community/notebooks-contrib/blob/main/community_tutorials_and_guides/weather.ipynb
```